homeassistant/components/derivative/sensor.py

```diff
@@ -46,6 +47,7 @@
         vol.Optional(CONF_UNIT_PREFIX, default=None): vol.In(UNIT_PREFIXES),
         vol.Optional(CONF_UNIT_TIME, default="h"): vol.In(UNIT_TIME),
         vol.Optional(CONF_UNIT): cv.string,
+        vol.Optional(CONF_TIME_WINDOW): vol.Coerce(float),
```

Comment:
I think that we should use `cv.time_period` ?

Reply:
Renamed :)